ansible_collections/arista/avd/roles/eos_designs/templates/network_services/vtep-logic.j2

```diff
@@ -1,30 +1,32 @@
 {# Setting evpn_rd_admin_subfield #}
 {% set network_services_data.evpn_rd_admin_subfield = switch.router_id %}
-{% if evpn_rd_type is defined and evpn_rd_type is not none %}
-{%     if evpn_rd_type.admin_subfield is arista.avd.defined %}
-{%         if evpn_rd_type.admin_subfield == "vtep_loopback" %}
-{%             set network_services_data.evpn_rd_admin_subfield = switch.vtep_ip %}
-{%         elif evpn_rd_type.admin_subfield == "bgp_as" %}
-{%             set network_services_data.evpn_rd_admin_subfield = switch.bgp_as %}
-{%         elif evpn_rd_type.admin_subfield | ansible.netcommon.ipaddr %}
-{%             set network_services_data.evpn_rd_admin_subfield = evpn_rd_type.admin_subfield %}
-{%         elif evpn_rd_type.admin_subfield is number and
-                evpn_rd_type.admin_subfield >= 0 and
-                evpn_rd_type.admin_subfield <= 4294967295 %}
-{%             set network_services_data.evpn_rd_admin_subfield = evpn_rd_type.admin_subfield %}
-{%         endif %}
+{% if overlay_rd_type.admin_subfield is arista.avd.defined %}
```

Comment:
important that you document this new variable name and make it fall back to the old one for now.

Reply:
maybe consider creating a separate logic for LER use case.

Reply:
Fixed in PR #1488